website/src/data/users.tsx

```diff
@@ -1301,7 +1301,15 @@ const Users: User[] = [
     website: 'https://haochenqi.com/',
     source: 'https://github.com/HaochenQ/Haochen-Blog',
     tags: ['personal', 'opensource']
-  }
+  },
+  {
+    title: 'Drayman',
+    description: 'Server-side component framework',
+    preview: require('./showcase/drayman.png'),
```

Comment:
We need exactly screenshot of your site (_I mean, with navbar_), look at screenshots of other sites as an example.

Reply:
Done